src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -3759,14 +3761,14 @@ instruct vsignum_reg(vReg dst, vReg zero, vReg one, vRegMask_V0 v0) %{
 
 // -------------------------------- Reverse Bytes Vector Operations ------------------------
 
-instruct vreverse_bytes_masked(vReg dst, vReg src, vRegMask_V0 v0) %{
-  match(Set dst (ReverseBytesV src v0));
-  format %{ "vreverse_bytes_masked $dst, $src, v0" %}
+instruct vreverse_bytes_masked(vReg dst_src, vRegMask_V0 v0) %{
+  match(Set dst_src (ReverseBytesV dst_src v0));
+  format %{ "vreverse_bytes_masked $dst_src, $dst_src, v0" %}
```

Comment:
Nit: I think we can use something more accurate like `vrev8.v` as the opcode name in format. That will be consistent with the RVV spec. Also I suggest `v0.t` instead of `v0` or `$v0` as the mask (Might deserve a separate PR for cleaning up other existing instructs). Similar for other newly added instructs.

Reply:
Sure, let me do it later, tracked by https://bugs.openjdk.org/browse/JDK-8332030.